Sources/FoundationInternationalization/Formatting/Duration+UnitsFormatStyle.swift

```diff
@@ -602,9 +630,6 @@ extension Duration.UnitsFormatStyle {
     }
 }
 
-@available(macOS 13.0, iOS 16.0, tvOS 16.0, watchOS 9.0, *)
-extension UATimeUnitStyle : Codable, Hashable {}
```

Comment:
This is safe because `UATimeUnitStyle` is from `FoundationICU` so this extension is not "public". This resolves a warning about this retroactive conformance by implementing the `Codable`/`Hashable` conformances ourselves above